### PR TITLE
feat(auto-session): remove customized options

### PR DESCRIPTION
**Pull request by Saafo**
add [recommended options](https://github.com/rmagatti/auto-session#recommended-sessionoptions-config) for auto-session, mainly to remember which buffers I've opened last time

### CODE REVIEW

**Review by Jint-lzxy — CHANGES_REQUESTED** [review]
All native options should be set in `lua/core/options.lua`:
https://github.com/ayamir/nvimdots/blob/a014422d0184c6009f9c464515fe898387a8f104/lua/core/options.lua#L61

**Comment by CharlesChiuGit** [comment]
in this case, i think deleting that settings would be nice.

**Comment by Saafo** [comment]
Ok, I deleted the option.

**Review by CharlesChiuGit — APPROVED** [review]
lgtm

File: lua/core/options.lua [hunk 1]
```diff
@@ -58,7 +58,6 @@ local function load_options()
 		relativenumber = true,
 		ruler = true,
 		scrolloff = 2,
```
Comment (Jint-lzxy) [inline lua/core/options.lua:60]:
```suggestion
		scrolloff = 2,
		sessionoptions = "buffers,curdir,help,tabpages,winsize",
```

**Review by Jint-lzxy — APPROVED** [review]
LGTM. Thanks for the contribution 🍿

**Pull request merged**